crates/miden-lib/asm/miden/active_account.masm

```diff
@@ -337,28 +339,63 @@ export.get_initial_item
     # => [INIT_VALUE]
 end
 
+#! Sets an item in the account storage.
+#!
+#! Inputs:  [name_id_prefix, name_id_suffix, VALUE]
+#! Outputs: [OLD_VALUE]
+#!
+#! Where:
+#! - name_id_{prefix, suffix} are the prefix and suffix felts of the name identifier, which are
+#!   the first two felts of the hashed slot name.
+#! - VALUE is the value to set.
+#! - OLD_VALUE is the previous value of the item.
+#!
+#! Panics if:
+#! - a slot with the provided name ID does not exist in account storage.
+#! - the invocation of this procedure does not originate from the native account.
+#! - the native account is a faucet and the provided name ID points to the reserved faucet storage slot.
+#!
+#! Invocation: exec
+export.set_item
```

Comment:
Why did we need to add this procedure here? We already have `set_item` in `native_account` - and the idea of the `active_account` module is that it contains procedures that are applicable to both native and foreign accounts.